front/assets/style/base.scss

```diff
@@ -102,11 +107,20 @@
 
 /* dark theme styles */
 .theme--dark.v-btn {
-  background: $primary-color !important;
-  color: white !important;
+  background-color: $btn-default-background !important;
+  padding: 11px 24px !important;
+  color: $btn-default-color !important;
+  font-family: Avenir LT Std;
 
   &:hover {
-    background: $primary-color !important;
+    height: 46px !important;
+  }
+  &:focus {
+    height: 46px !important;
+    background-color: $btn-focus-background !important;
+  }
+  &.v-btn--disabled:not(.v-btn--flat):not(.v-btn--text):not(.v-btn--outlined) {
+    background-color: $btn-disabled-background !important;
```

Comment:
disabled color should be set just by setting var
https://vuetifyjs.com/en/customization/sass-variables/

    'buttons': (
      'disabled': rgba(map-get($shades, 'black'), 0.26),